com/edocti/jintro/Exercise6.java

```diff
@@ -0,0 +1,211 @@
+import java.lang.Math;
+
+public class Exercise6 {
+
+	// getName
+	public static String getName(String student) {
+		String[] lines = student.split("\n");
+		if (lines == null || lines.length == 0) {
+			return "You must insert the students !";

```

Comment:
Not a good return value; you either throw an exception (which we didn't learn yet) or simply return null